tutorials/application_design/py/subscriber.py

```diff
@@ -45,71 +38,110 @@ def __init__(
     ):
         self._metrics_reader = metrics_reader
         self._transit_reader = transit_reader
-        self._dashboard_data: typing.Dict[
-            dds.InstanceHandle, DashboardItem
-        ] = dict()
 
     def __repr__(self):
-        return f"Dashboard({self._metrics_reader=}, {self._transit_reader=}, {self._dashboard_data=})"
+        return f"Dashboard({self._metrics_reader=}, {self._transit_reader=})"
```

Comment:
Does the string representation of the DataReader produce anything useful?

Can you show in the PR an example output of a running dashboard?

Reply:
I'm not sure if I'd call it useful, as it prints the pointer-like repr of the entities.

```
❯ python3 -m subscriber
Running dashboard: dashboard=Dashboard(self._metrics_reader=<rti.connextdds.DataReader object at 0x7f360e5e2bb0>, self._transit_reader=<rti.connextdds.DataReader object at 0x7f360e5324f0>)
[[ DASHBOARD: 2024-09-30 10:02:21.532963 ]]
Online vehicles: 0

Offline vehicles: 0

[INFO] Vehicle PALN6ZXN71WG318XU is enroute to Coord(lat=-46.10992285726171, lon=-23.36607730023902) from Coord(lat=-17.29690932734892, lon=36.59023701656847)

# ...

[[ DASHBOARD: 2024-09-30 10:02:26.533522 ]]
Online vehicles: 1
Vehicle PALN6ZXN71WG318XU:
  Fuel updates: 6
  Last known destination: Coord(lat=-46.10992285726171, lon=-23.36607730023902)
  Last known fuel level: 81.3764490923471

Offline vehicles: 0

# ...

[INFO] Vehicle PALN6ZXN71WG318XU has arrived at its destination in Coord(lat=29.106641859065363, lon=41.93699410392373)

[INFO] Vehicle PALN6ZXN71WG318XU is enroute to Coord(lat=9.017138169227357, lon=36.99741430219118) from Coord(lat=29.106641859065363, lon=41.93699410392373)

[INFO] Vehicle PALN6ZXN71WG318XU is enroute to Coord(lat=9.017138169227357, lon=36.99741430219118) from Coord(lat=-28.712438759150604, lon=32.36664575065243)

[[ DASHBOARD: 2024-09-30 10:02:46.534679 ]]
Online vehicles: 0

Offline vehicles: 1
Vehicle PALN6ZXN71WG318XU
```

Reply:
I don't think that repr does anything useful then. Can you remove it or e.g. assign a name to the dashboard instead and print the name?